core/examples/themes.js

```diff
@@ -5,98 +5,116 @@ App.themes = {
     "local": [{
         "children": [{
             "isExpanded": true,
-            "isInternalWMS": true,
-            "name": "Theme 1 - Physical",
+            "name": "OSM function",
             "isBaseLayer": false,
+            "id": 68,
+            "isInternalWMS": true,
             "children": [{
-                "name": "Features",
-                "isExpanded": true,
-                "children": [{
-                    "name": "drain_fn",
-                    "legend": true,
-                    "isChecked": true
-                }, {
-                    "name": "drainage",
-                    "legend": true,
-                    "isChecked": true,
-                    "legendRule": "Water"
-                }, {
-                    "name": "Others",
-                    "children": [{
-                        "name": "rail",
-                        "legend": true,
-                        "isChecked": true,
-                        "minResolutionHint": 0.00005,
-                        "maxResolutionHint": 0.002
-                    }, {
-                        "name": "road",
-                        "isChecked": true,
-                        "legend": true,
-                        "legendRule": "Roads"
-                    }]
-                }]
-            }, {
-                "name": "fedlimit",
+                "name": "osm_time",
+                "icon": "http://geomapfish.demo-camptocamp.com/pgiraud/wsgi/mapserv_proxy?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetLegendGraphic&LAYER=osm_time&FORMAT=image/png&TRANSPARENT=TRUE&RULE=Dans les temps",
```

Comment:
Shouldn't you rather use http://geomapfish.demo-camptocamp.com/1.6/mapserv_proxy ?